core/templates/dev/head/base-components/base-content.directive.html

```diff
@@ -66,10 +66,5 @@
       Dev Mode
     </div>
 
-    <a ng-if="$ctrl.siteFeedbackFormUrl" ng-href="<[$ctrl.siteFeedbackFormUrl]>" target="_blank" rel="noopener"
-       class="oppia-site-feedback oppia-transition-200">
```

Comment:
Please remove `oppia-site-feedback` from CSS (if it is not used elsewhere)

Reply:
@vojtechjelinek , This class is used in the site-feedback link in the exploration-editor. 
The problem with removing the site-feedback-link in the exploration-editor is that the exploration-editor has no footer. 
Should we still remove the feedback link in the exploration-editor?

Reply:
Hmmm, since we want to get rid of it because it makes the page hard to use on mobile try to test the editor on mobile and see and make screenshots of how it affects the UI.